### DISCUSSION TITLE
Are there any plans to port this spreadsheet for Vue/Nuxt too

### DISCUSSION THREAD

**Discussion by rmrbytes** [iddan/react-spreadsheet#290] [Ideas]
Hello:

I find this a wonderfully simple spreadsheet yet useful. For a project that is being done in Nuxt, this would be a good component to use. Have some folks forked it and working on it?

Thanks

**Comment by iddan** [comment]
Currently, I am not actively working on it. I do think it makes a lot of sense to have a shared core and use it for multiple frameworks (in a similar fashion to TanStack's Table) but it's not the focus of this project. I won't block changes that will make this possible though.
